crates/cast/bin/cmd/storage.rs

```diff
@@ -127,12 +127,10 @@ impl StorageArgs {
             }
         }
 
-        if !self.etherscan.has_key() {
```

Comment:
This check does not match the later api_key assignment.
In my case, this check fails, but config.get_etherscan_api_key works.